getting-started-2.0.md

```diff
@@ -0,0 +1,155 @@
+
+The easiest way to publish content is to have [_Helix Pages_](https://www.project-helix.page/) render and deliver a Markdown file in your GitHub repository:
+
+1. Click the _Create new repository_ button to set up a new public [GitHub repository](https://github.com/new/).
+   <br>![](./dummy.png "TODO: Create a repository")
+1. Click the _Create new file_ button in your GitHub repository to add a new file called `index.md`. This will be your homepage.
+   <br>![](./dummy.png "TODO: Create an index.md")
+1. Add some great Markdown content. Feel free to use [this example](https://raw.githubusercontent.com/adobe/helix-example/master/index.md "example index.md") for starters.
+1. Click the _Commit changes_ button.
+1. Go to `https://<my-repo>-<my_org>.project-helix.page/` to admire your work. Replace `<my-repo>` and `<my-org>` with your actual repository and organization names:
+   <br>![](./dummy.png "TODO: Helix Pages rendering your content")
+1. **Congratulations!** 🎉 You have just published your first page using _Helix Pages_!
+
+> **Hint:** You can add as many Markdown files as you like, and link them in Markdown.
+
+> **Note:** If your GitHub organization contains one or more dashes, add an extra dash between the repository and organization name in the _Helix Pages_ URL. For example, if your organization is called `my-org`, and your repository is `test`, the corresponding _Helix Pages_ URL would be `https://test--my-org.project-helix.page/`.
+
+## Step 2: Customize header and footer
+
+_Helix Pages_ can render a header and footer based on Markdown content in your GitHub repository:
+
+1. Add a file called `header.md` to your GitHub repository.
+   <br>![](./dummy.png "TODO: Create a header.md")
+1. Add a logo, and maybe a list of other links you would like to show in the header navigation. Here's some [inspiration](https://raw.githubusercontent.com/adobe/helix-example/master/header.md "example header.md").
+1. Click the _Commit changes_ button.
+1. If you added a logo in step 2, upload that now.
+   <br>![](./dummy.png "TODO: Upload logo")
+1. Add a file called `footer.md` to your GitHub repository.
+1. Add some content you would like to be shown in the footer. Here's a [proposal](https://raw.githubusercontent.com/adobe/helix-example/master/footer.md "example footer.md").
+1. Click the _Commit changes_ button.
+1. Reload `https://<my-repo>-<my_org>.project-helix.page/` to see header and footer in action.
+   <br>![TODO: Helix Pages rendering your header and footer](./dummy.png)
+
+## Step 3: Add your own HTML files
+
+> You'll need:
+> - basic HTML knowledge
+
+If you have existing HTML files or snippets that you wish to reuse, you can simply add them to your GitHub repository and have _Helix_Pages_ merge and deliver them for you:
+
+1. Add a `static.html` file to your GitHub repository. The file name can be anything, just make sure it has a `.html` extension.
```

Comment:
please don't use the name `static`. it has too many connotations already. what about `about.html` ?